src/pages/home/home.html

```diff
@@ -6,13 +6,19 @@
       <ion-title>
         <img src="./assets/images/header-clean.png" class="logo" alt="infomobi">
       </ion-title>
-      <ion-buttons end *ngIf="settings && settings.youtubeChannelId">
-        <button ion-button block (click)="pushVideosPage(settings.youtubeChannelId)">
+      <ion-buttons end>
+        <button ion-button block (click)="pushVideosPage(settings.youtubeChannelId)" *ngIf="settings && settings.youtubeChannelId">
           <div>
-            <ion-icon name="desktop"></ion-icon>
-            <label>TV</label>
+            <ion-icon name="logo-youtube"></ion-icon>
+            <label>{{'Videos' | translate}}</label>
```

Comment:
No need to apply css to reduce label font size ?

Reply:
![image](https://user-images.githubusercontent.com/32454870/53641258-1f5de780-3c0d-11e9-9e84-e72876ceacda.png)
looks like a good font-size, should reduce?